CMPUT301F23T07/pocket-inventory
# Home Page List and Add Item Activity

This PR adds the home page (developed by @Shikhararora19) and the ability to add items (@SheldonRoberts ).

The home page list uses a recycler view. It's currently not styled. The layout is designed to assist with debugging while other features are being developed. 

Note: The Total Value feature is not fully functioning in this version. It's there as a placeholder.
Note: The items are currently added to a list. This will eventually be replaced by the database

How to test it:
```
git fetch origin
git switch -c add-item-feature origin/add-item-feature
```
Then run the app in Android Studio. Make sure you can add an item, and view it's details after hitting submit.

Here are some images of what it should look like:
![Screenshot from 2023-11-01 13-30-33](https://github.com/CMPUT301F23T07/pocket-inventory/assets/52471071/339c06ed-64e6-40b1-9d88-0687fd7d2b13)
![Screenshot from 2023-11-01 13-31-28](https://github.com/CMPUT301F23T07/pocket-inventory/assets/52471071/3e663dcb-71ea-4c69-a931-2a894b107b87)
![Screenshot from 2023-11-01 13-31-38](https://github.com/CMPUT301F23T07/pocket-inventory/assets/52471071/be37a43e-9738-469f-a632-e3acc471b821)
